include/cantera/base/Array.h

```diff
@@ -67,6 +67,10 @@ class Array2D
         :  m_data(0), m_nrows(m), m_ncols(n) {
         m_data.resize(n*m);
         std::fill(m_data.begin(), m_data.end(), v);
+
+
+//std::cout << "array costructor 1" << m_nrows << "   " << m_ncols  << std::endl;
+
```

Comment:
Please avoid adding commented-out code, especially to otherwise-unmodified files.